Applications/shapeworks/MeshCommands.cpp

```diff
@@ -1068,26 +1069,64 @@ bool WarpMesh::execute(const optparse::Values &options, SharedCommandData &share
     return false;
   }
 
+  bool multiple = static_cast<bool>(options.get("multiple"));
+
   try {
     Mesh inputMesh(inputMeshFilename);
 
     std::vector<std::string> paths;
     paths.push_back(inputPointsFilename);
-    paths.push_back(targetPointsFilename);
-    ParticleSystem particlesystem(paths);
-    Eigen::MatrixXd allPts = particlesystem.Particles();
-    Eigen::MatrixXd staticPoints = allPts.col(0);
-    Eigen::MatrixXd movingPoints = allPts.col(1);
-    int numParticles = staticPoints.rows() / 3;
-    staticPoints.resize(3, numParticles);
-    staticPoints.transposeInPlace();
-    movingPoints.resize(3, numParticles);
-    movingPoints.transposeInPlace();
-
-    MeshWarper warper;
-    warper.set_reference_mesh(inputMesh.getVTKMesh(), staticPoints);
-    Mesh output = warper.build_mesh(movingPoints);
-    output.write(targetMeshFilename);
+    if(multiple){
```

Comment:
add spaces to more closely match the style in the rest of the file: `if (multiple) {` (it also seems the rest of the file inconsistently puts open braces on the same line or a separate line, so this example uses same line, but it does use spaces)

Reply:
got it

Reply:
Fixed